Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -365,39 +372,148 @@ def start_chrome_headless(chrome_port, chrome_binary=CHROME_EXE, user_options=""
     return None, None
 
 
-def terminate_chrome(browser):
-    global CHROME_PROCESS
-    demisto.debug(f'terminate_chrome, {CHROME_PROCESS=}')
+def terminate_chrome(chrome_port=None, killall=False):
+    processes = subprocess.check_output(['ps', 'auxww'], stderr=subprocess.STDOUT, text=True).splitlines()
+    chrome_renderer_identifiers = ["--type=renderer"]
+    chrome_identifiers = ["chrom", "headless"] if killall else ["chrom", "headless", f"--remote-debugging-port={chrome_port}"]
+    process_in_list = [process for process in processes
+                       if all(identifier in process for identifier in chrome_identifiers)
+                       and not any(identifier in process for identifier in chrome_renderer_identifiers)]
```

Comment:
Are you sure it's needed?
isn't `--remote-debugging-port={chrome_port}` (or `--remote-debugging-port=` in the killall scenario) enough?

Reply:
Why was this marked as "resolved"?
Are you sure there are chrome processes with `--remote-debugging-port={chrome_port}` and `--type=renderer` ?